src/main/java/entities/DesignableMaze.java

```diff
@@ -12,4 +16,65 @@ public void placeWall(int row, int col) {
     public void deleteWall(int row, int col) {
         this.state[row][col] = Maze.ENCODING.get("empty");
     }
-}
+
+    public void placeStart(int row, int col) {
+        for (int i = 0; i < this.getNumRow(); i++) {
+            for (int j = 0; j < this.getNumCol(); j++) {
+                if (this.state[i][j] == 'S'){
```

Comment:
reference the encoding instead of writing 'S'

Reply:
Fixed